src/scripts/convert_lovd2.php

```diff
@@ -0,0 +1,1346 @@
+require_once ROOT_PATH . 'class/progress_bar.php';
+require_once ROOT_PATH . 'class/soap_client.php';
+
+// Global for storing warning messages during conversion.
+$_WARNINGS = array();
+
+// Array of field names to ignore in input.
+$aIgnoredFields = array_flip(array('ID_sort_'));
+
+// Links between LOVD2-LOVD3 fields, with optional conversion function. Format:
+// LOVD2_field => array(LOVD3_section, LOVD3_field, Conversion_function)
+// Where LOVD2_field is an LOVD2 field name as it occurs in the input file,
+// LOVD3_section is a output section name as defined as key in the
+// $aImportSections variable, LOVD3_field is an LOVD3 field name and
+// Conversion_function is an optional name of a function taking a LOVD2 field
+// value as a string as argument and returning LOVD3 field value as a string.
+$aFieldLinks = array(
+    // This field maps to either VOT/Published_as or VOG/Published_as (handled later specifically).
+    'Variant/DNA_published' =>          array('vot',        'VariantOnTranscript/Published_as'),
+    'Variant/DBID' =>                   array('vog',        'VariantOnGenome/DBID',         'lovd_convertDBID'),
+    'Variant/Restriction_site' =>       array('vog',        'VariantOnGenome/Restriction_site'),
+    'Variant/Remarks' =>                array('vog',        'VariantOnGenome/Remarks'),
+    'Variant/Genetic_origin' =>         array('vog',        'VariantOnGenome/Genetic_origin'),
+    'Variant/Detection/Technique' =>    array('screening',  'Screening/Technique',          'lovd_convertScrTech'),
+    'Variant/Reference' =>              array('vot',        'VariantOnTranscript/Reference','lovd_convertReference'),
+    'Patient/Patient_ID' =>             array('individual', 'Individual/Lab_ID'),
+    'Patient/Reference' =>              array('individual', 'Individual/Reference',         'lovd_convertReference'),
+    'Patient/Gender' =>                 array('individual', 'Individual/Gender',            'lovd_convertGender'),
+    'Patient/Times_Reported' =>         array('individual', 'panel_size'),
+    'Patient/Phenotype_2' =>            array('phenotype',  'Phenotype/Additional'),
+    // Note that 'Patient/Phenotype/Inheritance' automatically maps to Phenotype/Inheritance as well.
+    'Patient/Occurrence' =>             array('phenotype',  'Phenotype/Inheritance',        'lovd_convertInheritance'),
+    'Patient/Mutation/Origin' =>        array('vog',        'VariantOnGenome/Genetic_origin',   'lovd_convertOrigin'),
+    'ID_pathogenic_' =>                 array('vog',        'effectid'),
+    'ID_status_' =>                     array('vog',        'statusid',                     'lovd_convertStatus'),
+    'ID_variant_created_by_' =>         array('vog',        'created_by',                   'lovd_convertCuratorID'),
+    'variant_created_date_' =>          array('vog',        'created_date'),
+    'ID_variant_edited_by_' =>          array('vog',        'edited_by',                    'lovd_convertCuratorID'),
```

Comment:
Just agreed to this change:
- The `edited_by` should not be overwritten with the fixed curator ID, unless the `edited_by` is actually filled in with something in the LOVD2 file.

An non-empty `edited_by` field results in the entry being imported in LOVD3 as "edited" whereas in LOVD2 it has never been edited.